code/modules/shuttle/custom_shuttle/shuttle_creator_eye.dm

```diff
@@ -0,0 +1,52 @@
+//===============Camera Eye================
+/mob/camera/ai_eye/remote/shuttle_creation
+	name = "shuttle holo-drone"
+	/*icon = 'icons/obj/mining.dmi'*/
+	icon_state = "construction_drone"
+	visible_icon =  FALSE
+	acceleration = 0
+	var/turf/source_turf
+	var/max_range = 12
+
+/mob/camera/ai_eye/remote/shuttle_creation/Initialize(mapload)
+	. = ..()
+	setLoc(get_turf(source_turf))
+
+/mob/camera/ai_eye/remote/shuttle_creation/update_remote_sight(mob/living/user)
+	user.sight = BLIND|SEE_TURFS
+	return TRUE
+
+/mob/camera/ai_eye/remote/shuttle_creation/relaymove(mob/user, direct)
+	dir = direct //This camera eye is visible as a drone, and needs to keep the dir updated
+	var/initial = initial(sprint)
+	var/max_sprint = 50
+
+	if(cooldown && cooldown < world.timeofday) // 3 seconds
+		sprint = initial
+
+	for(var/i = 0; i < max(sprint, initial); i += 20)
+		var/turf/step = get_turf(get_step(src, direct))
+		if(step && can_move_to(step))
+			setLoc(step)
+
+	cooldown = world.timeofday + 5
+	if(acceleration)
+		sprint = min(sprint + 0.5, max_sprint)
+	else
+		sprint = initial
```

Comment:
this is a full copy of remote/relaymove so far as I can tell. why? what's going on?